client/src/Component/videos/videoForm.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import {backEndApi } from "../../config/config.js";
+
+const VideoForm = ({ handleAddVideo }) => {
+  const [title, setTitle] = useState("");
+  const [url, setUrl] = useState("");
+
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    console.log("Form submitted");
+    if (title.trim() === "" || !isValidUrl(url)) {
+      return;
+    }
+
+    try {
+      const response = await fetch(`${backEndApi}`, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ title, url }),
+      });
+
+      if (response.ok) {
+        setTitle("");
+        setUrl("");
+      } else {
+        console.error("Error adding video:", response.statusText);
+      }
+    } catch (error) {
+      console.error("Error adding video:", error);
+    }
+  };
+
+  const isValidUrl = (url) => {
+    return url.trim() !== "";
```

Comment:
This function seems a bit misleadingly named - it's not really checking whether its parameter is a valid URL... Can you think how to either better name the function, or better make it do what its name suggests?